bench/workgen/workgen_int.h

```diff
@@ -55,20 +55,24 @@ struct WorkgenTimeStamp {
     WorkgenTimeStamp() {}
 
     static uint64_t get_timestamp_lag(double seconds) {
-        uint64_t start_time;
-        workgen_clock(&start_time);
-
-        return (ns_to_us(start_time) - secs_us(seconds));
+        return (get_timestamp() - secs_us(seconds));
     }
 
     static void sleep(double seconds) {
         usleep(ceil(secs_us(seconds)));
     }
 
     static uint64_t get_timestamp() {
```

Comment:
I don't understand the purpose of that function, should be called, get_epoch, where is `last_ts` used?

Reply:
Which function? The get_timestamp implementation or the usage within get_timestamp_lag?

Reply:
Sorry, now things make sense. The function `get_timestamp` just gives back a valid ts.